### ISSUE TITLE
Get `make test` passing on current develop HEAD

### ISSUE THREAD

**Issue by angus-lherrou** [clamsproject/mmif#62]
Tasks:

- [x] Find all roadblocks to passing
- [x] Append them here
- [x] Fix the following from `pytype`, which is being discussed in #61 (Note: if we merge #60, this is moot): 
```
unsupported operand type(s) for item assignment: 'AnnotationProperties' [unsupported-operands]
  No attribute '__setitem__' on AnnotationProperties
```
- [x] Fix:
```
File "/Users/stygg/clamsproject/mmif/mmif-python/mmif/serialize/__init__.py", line 6, in <module>: Name 'annotation' is not defined [name-error]
File "/Users/stygg/clamsproject/mmif/mmif-python/mmif/serialize/__init__.py", line 6, in <module>: No attribute '__all__' on module [attribute-error]
File "/Users/stygg/clamsproject/mmif/mmif-python/mmif/serialize/__init__.py", line 6, in <module>: Name 'view' is not defined [name-error]
File "/Users/stygg/clamsproject/mmif/mmif-python/mmif/serialize/__init__.py", line 6, in <module>: Name 'medium' is not defined [name-error]
```
- [x] Fix: won't-fix, `setup.py` must be out of pytype scope
```
File "/Users/stygg/clamsproject/mmif/mmif-python/setup.py", line 88, in <module>: No attribute 'sdist' on module 'setuptools.command' [module-attr]
File "/Users/stygg/clamsproject/mmif/mmif-python/setup.py", line 108, in <module>: No attribute 'setup' on module 'setuptools' [module-attr]
File "/Users/stygg/clamsproject/mmif/mmif-python/setup.py", line 117, in <module>: No attribute 'find_packages' on module 'setuptools' [module-attr]
```
- [x] Fix problems and PR into develop

Future:
- [ ] Test and fix additional problems for code in #60 before un-drafting

**Comment by angus-lherrou** [comment]
I'm working on this tomorrow morning

**Comment by angus-lherrou** [comment]
@keighrim any thoughts on the issue with setuptools in setup.py that pytype encounters above?

**Comment by angus-lherrou** [comment]
@keighrim pytype is passing for the `mmif` package itself now, but not for `setup.py` or `test_serialize.py`. Do we want to rewrite the Makefile to do `pytype ./mmif` instead of `pytype .`?

**Comment by keighrim** [comment]
@angus-lherrou That makes much more sense. I'll push a commit right now.

**Comment by keighrim** [comment]
a8859b5556cb40ad124fc90709988949a16747b7 should fix pytype scope.

**Comment by angus-lherrou** [comment]
Great, 0af6ae91c3b8ef54e3f9bb2dd9fbb8b0c12032a4 brings us to a working state.

Let me write some unit tests for `AnnotationProperties` and `MediumMetadata` and then I'll open a PR

**Comment by keighrim** [comment]
Closed via #63 .
